src/Components/Chatpage.tsx

```diff
@@ -4,21 +4,22 @@ import { Link } from 'react-router-dom';
 import EmojiPicker from 'emoji-picker-react';
 import InputEmojiWithRef from 'react-input-emoji';
 import { HubConnection, HubConnectionBuilder } from '@microsoft/signalr';
+import { toast } from 'react-toastify';
 var compare = ""
-var name = "";
+var chatImg = "";
```

Comment:
avoid using 'var'. Replace 'var' in the whole code base with let or const.
While doing this, learn the difference between them!

Reply:
Yes sir
I made the changes